modules/users/client/components/ProfileViewBasics.component.js

```diff
@@ -1,21 +1,174 @@
 import React from 'react';
-import { NamespacesConsumer } from 'react-i18next';
+import classnames from 'classnames';
+import { withNamespaces } from './utils/i18n-angular-load';
 import '@/config/lib/i18n';
 import PropTypes from 'prop-types';
+import * as languages from '@/config/languages/languages';
+import { hasConnectedAdditionalSocialAccounts, isWarmshowersId, socialAccountLink } from './utils/networks';
 
-export default function ProfileViewBasics({ profile }) {
-  return (<NamespacesConsumer ns="user-profile">{ t => (<>
+export function ProfileViewBasics({ t, profile }) {
+  const renderReplyData = (replyRate, replyTime) => (
+    <div className="profile-sidebar-section text-muted">
+      {replyRate &&
+      <span>
+        {t('Reply rate {{replyRate}}.', { replyRate: profile.replyRate })}
+      </span>}
+      {replyTime &&
+      <span>
+        <br/>
+        {t('Replies within {{replyTime, fromNow}}.', { replyTime: profile.replyTime })}
+      </span>}
+    </div>
+  );
+
+  const renderBirthdateAndGender = (birthdate, gender) => (
+    <div className="profile-sidebar-section" >
+      {birthdate && t('{{birthdate, age}} years', { birthdate: new Date(birthdate) })}
+      {(birthdate && gender) && <span>, </span>}
+      <span className={classnames({ 'text-capitalize': !birthdate })}>{t(gender)}.</span>
+    </div>
+  );
+
+  const renderMemberSince = () => (
     <div className="profile-sidebar-section">
       {t('Member since {{date, MMM Do, YYYY}}', { date: new Date(profile.created) })}
     </div>
+  );
+
+  const renderSeenOnline = () => {
     <div className="profile-sidebar-section">
       <span>
         {(profile.seen) ? t('Online {{date, fromNow}}', { date: new Date(profile.seen) }) : t('Online long ago')}
       </span>
+    </div>;
+  };
+
+  const renderLocationLiving = locationLiving => (
+    <div className="profile-sidebar-section">
+      <i className="icon-fw icon-building text-muted" />
+      {t('Lives in')}{' '}
```

Comment:
It would be nice to have `t('Lives in {{location}}', { location: locationLiving })`. Some languages don't keep this word order.

Reply:
Well, the `locationLiving={locationLiving}` as attribute of `Trans` seems to not be necessary, right? Works without.